model/fv_regional_bc.F90

```diff
@@ -6632,25 +6632,38 @@ end subroutine exch_uv
 !+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 !---------------------------------------------------------------------
 
-  subroutine get_data_source(source,regional)
+  subroutine get_data_source(data_source_fv3gfs,regional)
 !
 ! This routine extracts the data source information if it is present in the datafile.
 !
-      character (len = 80) :: source
-      integer              :: ncids,sourceLength
+      character (len=80) :: source      
       logical :: lstatus,regional
+      logical :: data_source_fv3gfs
```

Comment:
Consider adding `intent(out)` to `data_source_fv3gfs` and `intent(in)` to  `regional`.

Reply:
Completed.